gemc/5.12/rgb_fall2019.gcard

```diff
@@ -0,0 +1,113 @@
+<!-- GEMC 5.12 -->
+<gcard>
+
+	<!-- band detector upstream of the target -->
+	<detector name="experiments/clas12/band/band" factory="TEXT" variation="main"/>
+
+	<detector name="experiments/clas12/targets/target" factory="TEXT" variation="rgb_fall2019"/>
+	<detector name="experiments/clas12/targets/cad/"   factory="CAD"/>
+
+	<!-- central detectors 	-->
+	<detector name="experiments/clas12/bst/bst"               factory="TEXT"/>
+	<detector name="experiments/clas12/micromegas/micromegas" factory="TEXT"/>
+
+	<!--ctof, cnd  -->
+	<detector name="experiments/clas12/ctof/ctof"    factory="TEXT" variation="rga_fall2018"/>
+	<detector name="experiments/clas12/ctof/cad/"    factory="CAD"  variation="rga_fall2018"/>
+	<detector name="experiments/clas12/cnd/cnd"      factory="TEXT" variation="rga_fall2018"/>
+
+	<!--high threshold cherenkov -->
+	<detector name="experiments/clas12/htcc/htcc"  factory="TEXT" variation="rga_fall2018"/>
+	<detector name="experiments/clas12/htcc/cad/"  factory="CAD"  variation="rga_fall2018"/>
+
+	<!-- magnets -->
+	<detector name="experiments/clas12/magnets/solenoid" factory="TEXT" variation="rga_fall2018"/>
+	<detector name="experiments/clas12/magnets/cad/"     factory="CAD" />
+
+	<!-- FT/Beamline configuration -->
+	<detector name="experiments/clas12/ft/ft"              factory="TEXT" variation="rgb_spring2019"/>
+	<detector name="experiments/clas12/beamline/cad/"      factory="CAD"  variation="rgb_spring2019"/>
+	<detector name="experiments/clas12/beamline/beamline"  factory="TEXT" variation="rgb_spring2019"/>
+
+	<!-- Detectors in the forward carriage -->
+	<detector name="experiments/clas12/dc/dc"       factory="TEXT"/>
+	<detector name="experiments/clas12/ftof/ftof"   factory="TEXT" variation="rga_spring2018"/>
+	<detector name="experiments/clas12/ec/ec"       factory="TEXT" variation="rga_spring2018"/>
+	<detector name="experiments/clas12/pcal/pcal"   factory="TEXT" variation="rga_spring2018"/>
+	<detector name="experiments/clas12/ltcc/ltcc"   factory="TEXT" variation="rgb_spring2020"/>
```

Comment:
how can the variation be in the future?

Reply:
human error. correct value is rga_spring2018